source/ListActions.js

```diff
@@ -678,6 +705,9 @@
 			// Skip animation before render time
 			if (!this.$.client.hasNode()) { return; }
 			this.$.client.addRemoveClass('open', this.open);
+			if (this.allowBackKey && this.open) {
```

Comment:
Instead of handling this in `moon.ListActionsDrawer`, can we handle this in `moon.ListActions`'s `openChanged` instead? It seems like we're spreading parts the back-key support between two kinds here. This would allow `moon.ListActionsDrawer` to not require any knowledge of back-key support (and the resulting one-off declaration of related properties and events), as its open/close state is effectively controlled by `moon.ListActions`.